README.md

```diff
@@ -26,92 +25,54 @@ throughout its lifetime. It looks like this:
 	+---------------+
 	        |            <--- database specific calls
 	+---------------+
-	|  various dbs  |    <--- MySQL, Redis, MongoDB, FS, ...
+	|  various dbs  |    <--- MySQL, Redis, MongoDB, FS, …
 	+---------------+
 
-In addition, grouped datastores significantly simplify interesting data access
-patterns (such as caching and sharding).
-
+In addition, datastore adapters and collections significantly simplify creating
+interesting data access patterns such as caching, sharding and tiering.
 
-## About
+## Table of Contents
 
-### Install
+  * [Install](#install)
+  * [Usage](#usage)
+  * [Documentation](#documentation)
+  * [API](#api)
+  * [Maintainers](#maintainers)
+  * [Thanks](#thanks)
+  * [Contributing](#contributing)
+     * [Other Datastore Implementations](#other-datastore-implementations)
+  * [License](#license)
 
-From pypi (using pip):
+## Install
 
-	sudo pip install datastore
+From pypi (WARNING – This is currently outdated!):
 
-From pypi (using setuptools):
-
-	sudo easy_install datastore
+	pip install datastore
 
 From source:
 
-	git clone https://github.com/ipfs/py-datastore/
+	git clone https://github.com/ipfs/py-datastore.git
 	cd datastore
-	sudo python setup.py install
-
-### Subprojects
-
-
-* [datastore.aws](https://github.com/datastore/datastore.aws) - aws s3 implementation
-* [datastore.git](https://github.com/datastore/datastore-git) - git implementation
-* [datastore.mongo](https://github.com/datastore/datastore.mongo) - monogdb implementation
-* [datastore.memcached](https://github.com/datastore/datastore.memcached) - memcached implementation
-* [datastore.pylru](https://github.com/datastore/datastore.pylru) - pylru cache implementation
-* [datastore.redis](https://github.com/datastore/datastore.redis) - redis implementation
-* [datastore.leveldb](https://github.com/datastore/datastore.leveldb) - leveldb implementation
-
+	flit install
 
-### Documentation
+### Datastore Implementations
 
-The documentation can be found at:
-http://datastore.readthedocs.org/en/latest/
-
-### License
+Only the filesystem and the in-memory `DictDatastore` are included in this project.
+For other known implementation please see the links below.
```

Comment:
Did you mean: "For other known _implementations_ ....."?

Reply:
Yes, I did! Thank you!